PubSub/src/Connection/Grpc.php

```diff
@@ -439,4 +459,26 @@ private function buildPushConfig(array $pushConfig)
     {
         return $this->serializer->decodeMessage(new PushConfig(), $pushConfig);
     }
+
+    private function transformDuration($v)
+    {
+        if (is_string($v)) {
+            $d = explode('.', trim($v, 's'));
+            if (count($d) !== 2) {
+                return null;
+            }
+
+            $seconds = (int) $d[0];
+            $nanos = (int) $d[1];
```

Comment:
This would turn `3.5s` into 3 seconds and 5 nanoseconds. Should it be `500,000,000` nanoseconds?

Reply:
This should be fixed now. PTAL